index.js

```diff
@@ -152,6 +155,10 @@ app.post("/authorized_post_request", authMiddleWare, (req, res) => {
 
 app.use("/", authRouter);
 
+app.use("/events", eventRouter);
+
+app.use("/teams", teamRouter);
```

Comment:
This is a small recommendation. If you're going to attach a router directly to `/` route, then put it as the last one. The reason for this is that if by any chance you have a route like `/:id` in the auth router, it will catch all the requests sent to other routers because its basically `/anything`